versioned_docs/version-2.x/magento2/search-engine.mdx

```diff
@@ -25,9 +25,9 @@ platforms:
 
 <SinceVersion tag="2.15" />
 
-Front-Commerce supports the native Magento API for search and category listing
-pages. It means that any Magento search module compatible with its GraphQL API
-will be used in your Front-Commerce project without any specific work.
+Front-Commerce supports the native Magento API for category listing pages. It
+means that any Magento search module compatible with its GraphQL API will be
+used in your Front-Commerce project without any specific work.
```

Comment:
**issue:** this is incorrect. Magento2 **has** native search supported.

See https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/639ddf71ed07c35040768b4c12de9225b638d911/src/server/modules/magento2/catalog/search/resolvers.js#L22

**suggestion:** revert this change